src/App.tsx

```diff
@@ -22,18 +22,106 @@ class App extends React.Component<{}, State> {
     ],
   };
 
-  render() {
-    const { images } = this.state;
+  const [step, setStep] = useState(3);
+  const [frameSize, setFrameSize] = useState(3);
+  const [itemWidth, setItemWidth] = useState(130);
+  const [animationDuration, setAnimationDuration] = useState(1000);
+  const [infinite, setInfinite] = useState(false);
 
-    return (
-      <div className="App">
-        {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
+  const handleStep = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setStep(+event.target.value);
+  };
+
+  const handleFrameSize = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setFrameSize(+event.target.value);
+  };
+
+  const handleItemWidth = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setItemWidth(+event.target.value);
+  };
+
+  const handleAnimationDuration = (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    setAnimationDuration(+event.target.value);
+  };
 
-        <Carousel />
+  const handleInfinite = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setInfinite(!!event.target.checked);
+  };
+
+  return (
+    <div className="App">
+      <h1 className="App__title" data-cy="title">
+        Carousel with
+        {' '}
+        {state.images.length}
+        {' '}
+        images
+      </h1>
```

Comment:
```suggestion
      <h1 className="App__title" data-cy="title">
        {`Carousel with ${state.images.length} images`}
      </h1>
```
use template strings